ft-fine-tune-bert/fine-tune-unittests.py

```diff
@@ -0,0 +1,118 @@
+            ],
+         ]
+
+        with open("6_genre_clean_training_data_small_unittests.txt", "w") as output:
+            for row in self.train_data:
+                for col in row: 
+                    output.write(str(col) + '\n')
+
+        with open("6_genre_eval_data_small_unittests.txt", "w") as output:
+            for row in self.eval_data:
+                for col in row:
+                    output.write(str(col) + '\n')
+
+
+    def assertIsFile(self, path):
+        if not pl.Path(path).resolve().is_file():
+            raise AssertionError("File does not exist: %s" % str(path))
+    
+    def assertFileExist(self, path):
+        if not pl.Path(path):
+            raise ValueError(
+                "Cannot do evaluation without an evaluation data file. Either supply a file to --eval_data_file "
+                "or remove the --do_eval argument."
+            )
+    def assertDirExist(self, path):
+        if (
+            os.path.exists(path)
+        ):
+            raise ValueError(
+                f"Output directory ({path}) already exists and is not empty. Use --overwrite_output_dir to overcome."
+            )
+
+    # Test 1
+    def test_data_paths(self):
+        """Checks if input file existing"""
+        file = pl.Path("./6_genre_clean_training_data_small.txt")
+        self.assertIsFile(file)
+
+    # Test 2
+    def test_data_paths(self):
+        """Checks if input file existing"""
+        file = pl.Path("./6_genre_eval_data_small.txt")
+        self.assertIsFile(file)
+
```

Comment:
self.assertIsFile(file) and self.assertDirExist(file) raise exceptions. 

Hence you have to use the unittest exception testing methods. See https://github.com/cnvrg/unittest-framework/blob/main/connectorUnittest/test_connector_factory.py for example.